modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -491,17 +572,14 @@
 			to_chat(user, span_warning("You cannot do multiple things at the same time!"))
 			return
 		in_use = TRUE
-		if(forge_temperature <= 50)
-			to_chat(user, span_warning("The temperature is not hot enough to start heating [glass_item]."))
-			in_use = FALSE
+		if(forge_temperature < MIN_FORGE_TEMP)
+			fail_message(user, "The temperature is not hot enough to start heating [glass_item].")
```

Comment:
To reduce repetition of the same words across multiple different fail messages, maybe you could make this into a define? I'm thinking something like this
```
#define TEMP_FAIL(target) "The temperature is not hot enough to start heating [target]."
fail_message(user, TEMP_FAIL(glass_item))
```